packages/hoppscotch-sh-admin/src/pages/teams/index.vue

```diff
@@ -13,24 +13,107 @@
 
       <div class="overflow-x-auto">
         <div
-          v-if="fetching && !error && teamList.length === 0"
+          v-if="fetching && !error && teamsList.length === 0"
           class="flex justify-center"
         >
           <HoppSmartSpinner />
         </div>
 
         <div v-else-if="error">{{ t('teams.load_list_error') }}</div>
 
-        <TeamsTable
-          v-else
-          :teamList="teamList"
-          @goToTeamDetails="goToTeamDetails"
-          @deleteTeam="deleteTeam"
-          class=""
-        />
+        <div v-else-if="teamsList.length == 0" class="px-2">
+          <p class="text-lg">
+            {{ t('teams.no_teams') }}
+          </p>
+        </div>
+
+        <div v-else>
+          <HoppSmartTable :list="teamsList">
+            <template #head>
+              <tr
+                class="text-secondary border-b border-dividerDark text-sm text-left bg-primaryLight"
+              >
+                <th class="px-6 py-2">{{ t('teams.id') }}</th>
+                <th class="px-6 py-3">{{ t('teams.name') }}</th>
+                <th class="px-6 py-2">{{ t('teams.members') }}</th>
+                <th class="px-6 py-2"></th>
+              </tr>
+            </template>
+            <template #body="{ list }">
+              <tr
+                v-for="team in list"
+                :key="team.id"
+                class="text-secondaryDark hover:bg-divider hover:cursor-pointer rounded-xl"
+              >
+                <td
+                  @click="goToTeamDetails(team.id)"
+                  class="py-4 px-3 max-w-50"
+                >
+                  <div class="flex">
+                    <span class="truncate">
+                      {{ team.id }}
+                    </span>
+                  </div>
+                </td>
+
+                <td
+                  @click="goToTeamDetails(team.id)"
+                  class="py-4 px-3 min-w-80"
+                >
+                  <span
+                    v-if="team.name"
+                    class="flex items-center ml-4 truncate"
+                  >
+                    {{ team.name }}
+                  </span>
+                  <span v-else class="flex items-center ml-4">
+                    {{ t('teams.unnamed') }}
+                  </span>
+                </td>
+
+                <td @click="goToTeamDetails(team.id)" class="py-4 px-3">
+                  <span class="ml-7">
+                    {{ team.members?.length }}
+                  </span>
+                </td>
+
+                <td>
+                  <div class="relative">
```

Comment:
```suggestion
              <tr
                v-for="team in list"
                :key="team.id"
                class="text-secondaryDark hover:bg-divider hover:cursor-pointer rounded-xl"
                @click="goToTeamDetails(team.id)"
              >
                <td class="py-4 px-3 max-w-50">
                  <div class="flex">
                    <span class="truncate">
                      {{ team.id }}
                    </span>
                  </div>
                </td>

                <td class="py-4 px-3 min-w-80">
                  <span
                    v-if="team.name"
                    class="flex items-center ml-4 truncate"
                  >
                    {{ team.name }}
                  </span>
                  <span v-else class="flex items-center ml-4">
                    {{ t('teams.unnamed') }}
                  </span>
                </td>

                <td class="py-4 px-3">
                  <span class="ml-7">
                    {{ team.members?.length }}
                  </span>
                </td>

                <td @click.stop>
                  <div class="relative">
```

we can use @click on the row instead of all <td> and can stop event propagation on the tippy using @click.stop